### PR TITLE
Update migrate_conn.py to have dry_run mode

### PR DESCRIPTION
**Pull request by frankcash**
Trigger the dag with params of `{"dry_run": true}` so instead of migrating the dag will only log the `connection_id`, `host`, and `login`

Screenshot 2023-04-25 at 11 46 17 AM

### CODE REVIEW

**Comment by frankcash** [comment]
Looks like on runtime 7.4.1 / airflow 2.5.2 the prints need to be `            print(f'connection_id: {conn[3]} host: {conn[6]} login: {conn[8]}')` although I just followed the specified keys in the original code

File: dags/v1_10to2x/migrate_conn.py [hunk 1]
```diff
@@ -21,33 +28,36 @@ def migrate_conns():
...
+                        "extra": str(conn[1])
+                    }
+            url = Variable.get("ASTRO_URL")
+            token = Variable.get("ASTRO_ACCESS_KEY")
+            headers = {'Content-type': 'application/json', 'Authorization': f'Bearer {token}', 'Accept': 'application/json'}
+            r = requests.post(f'{url}/api/v1/connections', data=json.dumps(formatted_conn), headers=headers)
+            print(conn[3])
+            print(r.text)
+        else:
+            print(f'connection_id: {conn[3]} host: {conn[5]} login: {conn[7]}')
```
Comment (odaneau-astro) [inline dags/v1_10to2x/migrate_conn.py]: Why not print the formatted_con object instead here?

File: dags/v1_10to2x/migrate_conn.py [hunk 2]
```diff
@@ -21,33 +28,36 @@ def migrate_conns():
...
+
+    print(f'{conn_list=}')
 
 with DAG(
     dag_id="migrate_conns",
     schedule_interval=None,
     start_date=datetime(2022, 1, 1),
     catchup=False,
     tags=["migration"],
-) as dag:
```
Comment (odaneau-astro) [inline dags/v1_10to2x/migrate_conn.py:48]: This may break in 1.x

Comment (frankcash) [reply dags/v1_10to2x/migrate_conn.py:48]: I have added this back, will push shortly.

**Comment by odaneau-astro** [comment]
not a fan of indexes in formatted_conn. we should extract the column names in conn_list and use that instead.
conn_list = [
        [getattr(c, column.name) for column in Connection.__mapper__.columns]
        for c in connections
    ]

**Comment by frankcash** [comment]
@odaneau-astro great idea, let's create a backlog ticket for it :p

**Review by odaneau-astro — APPROVED** [review]

**Pull request merged**